Update Order Page View.md (#167)
PrestaShop/prestashop-specs: back-office/orders/orders/Order Page View.md

The fields "account registered" and "total spent" are hidden.
Below the customer name, "Guest" in grey (Open sans semibold 14px #6C868E) is displayed.

The button to transform a guest into a customer is removed. The feature is still available on the customer page. The order is not the place to handle customer management. It was removed to not encourage the merchant to transform Guest to a Customer account by letting a button on the order page.

3. **Shipping and Invoice address**

Each address has **a button** to update or change it. The button opens a list to select to edit or update it.
Editing an address creates a new address and soft-deletes the old one. So, if the edited address is used by an old order, it will not be updated with the new address.

> [Edit/Change Adresse behaviors](https://invis.io/YKTGGFZAMCB#/386397109_Edit_Adress)

Reviewer: ```suggestion
[Edit/Change Addresse behaviors](https://invis.io/YKTGGFZAMCB#/386397109_Edit_Adress)
```

-   **Multi invoices behaviors**

When I add a product when an invoice is already generated, the merchant can choose to: 
- Offer free shipping for this product or apply the shipping cost
- Associate the product to an existing or new invoice

When the order has several invoices, it is possible to add an already existing product to the order if they are related to a different invoice.
There is only one possible price per product even if they are from different invoices. Therefore, if a product is added with a different price in another invoice, the price will be updated on the first invoice. To avoid confusion, a pop-up window will be displayed before the addition to confirm that the action may have an impact on the other order.

3. **Remove products from the order**

Deleting one or several products is very easy: a remove icon is available to remove one or several quantities of a paid product. 

When a product is removed from the order, a notification is displayed: "The item was successfully removed." in Admin.Notifications.Error.

> Then the product list is refreshed behind the loading spinner to check if any gift product has to be removed as well. The user is redirected to the page where the user removed the product. If the removed product was on the last page, the user is redirected to the second-last page.

Reviewer: ```suggestion
Then the product list is refreshed behind the loading spinner to check if any gift product has to be removed as well. The user is redirected to the page where he/she was when he/she removed the product. If the removed product was on the last page, the user is redirected to the second-last page.
```

Below the list, a drop list of all the status available is displayed to update the current status through the **button Update Status** next to it. Updating the order status will trigger all the actions set by the order status.

> On the bottom of the bock, there is the **Order comment**. The order comment saves a comment for the corresponding order. It is closed by default when it is empty and open when filled. To add a comment when it is empty, the user has to click on the + button and has to press the save button to save it.

Reviewer: ```suggestion
On the bottom of the block, there is the **Order comment**. The order comment saves a comment for the corresponding order. It is closed by default when it is empty and open when filled. To add a comment when it is empty, the user has to click on the + button and has to press the save button to save it.
```

5.  **Payment panel**

The payment section displays all the transactions made in the order from the payment module or the manual inputs.

The table shows **the date and hours, the payment method, the transaction ID, the amount with the currency, and the button detail.**

To fill a payment transaction, the user has **to select the date, one payment method from a droplist, the amount, and currency then press the add button. The amount** can be a **positive or negative value**.

**A red warning is shown** if the filled amount is inferior or superior to the order’s total comparing the total of the transactions and order’s total.

After adding, the user can display **all the details of the transaction by pressing the Detail button**. It will show below the arrow **the number, brand, expiration date, and owner of the credit card** for any payment module.

**If the order’s status** is considered **as paid then before adding a transaction** the user has to specify in **which generated invoices from a droplist** it will be added.

When the status accepted (waiting for payment) you can change the currency of the order with a drop list.

> If the shop has different currencies, a droplist of the shop's currencies is displayed to change the order currency. The change rate is based on the latest conversion rate save by the shop.

Reviewer: ```suggestion
If the shop has different currencies, a droplist of the shop's currencies is displayed to change the order currency. The change rate is based on the latest conversion rate saved in the shop.
```
